BaikalStandalone/Utils/config_manager.h

```diff
@@ -33,53 +33,45 @@ namespace Baikal
     class Renderer;
 }
 
-class ConfigManager
+enum class DeviceType
 {
-public:
-
-    enum DeviceType
-    {
-        kPrimary,
-        kSecondary
-    };
-
-    enum Mode
-    {
-        kUseAll,
-        kUseGpus,
-        kUseSingleGpu,
-        kUseSingleCpu,
-        kUseCpus
-    };
-
-    struct Config
-    {
-        DeviceType type;
-        std::unique_ptr<Baikal::Renderer> renderer;
-        std::unique_ptr<Baikal::SceneController<Baikal::ClwScene>> controller;
-        std::unique_ptr<Baikal::RenderFactory<Baikal::ClwScene>> factory;
-        CLWContext context;
-        bool caninterop;
-
-        Config() = default;
+    kPrimary = 0,
+    kSecondary
+};
 
-        Config(Config&& cfg) = default;
+enum class Mode
+{
+    kUseAll = 0,
+    kUseGpus,
+    kUseSingleGpu,
+    kUseSingleCpu,
+    kUseCpus
+};
 
-        ~Config()
-        {
-        }
-    };
+struct Config
+{
+    DeviceType type;
+    std::unique_ptr<Baikal::Renderer> renderer;
+    std::unique_ptr<Baikal::SceneController<Baikal::ClwScene>> controller;
+    std::unique_ptr<Baikal::RenderFactory<Baikal::ClwScene>> factory;
+    CLWContext context;
+    bool caninterop;
 
-    static void CreateConfigs(
-        Mode mode,
-        bool interop,
-        std::vector<Config>& renderers,
-        int initial_num_bounces,
-        int req_platform_index = -1,
-        int req_device_index = -1);
+    Config() = default;
 
-private:
+    Config(Config&& cfg) = default;
 
+    ~Config()
+    {
+    }
 };
 
+void CreateConfigs(
+    Mode mode,
+    bool interop,
+    std::vector<Config>& renderers,
+    int initial_num_bounces,
+    int req_platform_index = -1,
+    int req_device_index = -1);
+
 #endif // CONFIG_MANAGER_H
```

Comment:
Having that almost whole file is rewritten, let's switch to pragma once.

Reply:
Done